src/Administration/Resources/app/administration/src/app/asyncComponent/form/sw-datepicker/index.ts

```diff
@@ -6,6 +6,8 @@ import template from './sw-datepicker.html.twig';
  * @private
  * @status ready
  * @description Wrapper component for sw-datepicker and mt-datepicker. Autoswitches between the two components.
+ *
+ * @deprecated tag:v6.8.0 - Will be removed, use sw-datepicker-deprecated instead
```

Comment:
shouldn't they use the `mt-*` stuff instead? :thinking:

Reply:
You're correct, should be the mt components

Reply:
🤦🏻‍♂️ 🤦🏻‍♂️ 🤦🏻‍♂️ 🤦🏻‍♂️ 🤦🏻‍♂️ 
Thanks again!